app/uk/gov/hmrc/incorporatedentityidentification/repositories/JourneyDataRepository.scala

```diff
@@ -80,4 +81,5 @@ class JourneyDataRepository @Inject()(reactiveMongoComponent: ReactiveMongoCompo
 
 object JourneyDataRepository {
   val journeyIdKey: String = "_id"
+  val authInternalIdKey: String = "_authInternalId"
```

Comment:
"_id" is a specific key that mongo uses to store an ID which we are overriding with our journeyId so you don't need to add the underscore at the start of authInternalIdKey